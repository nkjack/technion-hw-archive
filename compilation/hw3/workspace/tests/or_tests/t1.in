int a(){
return 3;
}

void main(){
int c = a();
}
