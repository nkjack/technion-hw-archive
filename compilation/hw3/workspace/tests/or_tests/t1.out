---end scope---
a - 0 preconditions
---end scope---
main - 0 preconditions
c INT 0
---end scope---
print (STRING)->VOID 0
printi (INT)->VOID 0
a ()->INT 0
main ()->VOID 0
